src/validators/length-validator.js

```diff
@@ -4,7 +4,7 @@ class _MinLengthValidator {
     message = '"{field}" must be at least {params} in length';
     params = 1;
     validate(value) {
-        const valid = !!value && value.length >= this.params;
+        const valid = !value || (!!value && value.length >= this.params) ;
```

Comment:
@fisnikpolymathlabs is this not the same as:
`!value || value.length >= this.params`

Reply:
Yes it's the same, since "!value" assures us that "value" is defined and are safe if we check for length.